systemtest/src/main/java/io/strimzi/systemtest/resources/crd/kafkaclients/KafkaBasicExampleAdminClient.java

```diff
@@ -0,0 +1,258 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.systemtest.resources.crd.kafkaclients;
+
+import io.fabric8.kubernetes.api.model.batch.v1.JobBuilder;
+import io.strimzi.systemtest.Constants;
+import io.strimzi.systemtest.Environment;
+import io.strimzi.systemtest.resources.ResourceManager;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+import java.security.InvalidParameterException;
+import java.util.HashMap;
+import java.util.Map;
+
+public class KafkaBasicExampleAdminClient {
```

Comment:
The Admin client does many different things. Here you seem to be building an interface heavily based on a single use-case (topic management). I wonder if this is really useful long term as it might need to be redesigned in the future.

(If the answer is _we will rewrite it when really needed_, I'm fine with it.)

Reply:
It is not example client anymore. So maybe it should not refer to it in the class name?

Reply:
Some more javadoc would not hurt this class either.

Reply:
`(If the answer is we will rewrite it when really needed, I'm fine with it.)` - that might be the case, as until now, none needed Admin Client and create topics in bulk (although I have found 1 test, where KT are created en-mass).
Example in name - good point
Javadoc - I will add it

Thank you!